src/relation_engine_server/utils/arango_client.py

```diff
@@ -36,8 +36,9 @@ def run_query(query_text=None, cursor_id=None, bind_vars={}):
     else:
         method = 'POST'
         req_json['count'] = True
-        req_json['bindVars'] = bind_vars
         req_json['query'] = query_text
+        if bind_vars:
+            req_json['bindVars'] = bind_vars
```

Comment:
Mutable default args can bite you. maybe not in the case of a webserver but just want to be safe.